app/client/components/callCentreNumbers.tsx

```diff
@@ -15,6 +15,12 @@ const callCenterStyles = css({
   fontWeight: "normal"
 });
 
+export const ukPhoneNumber = "+ 44 (0) 330 333 6790";
```

Comment:
MINOR: Please could these share a common non-exported base part, like...
```suggestion
const ukPhoneCommonPart = "330 333 6790";
export const ukPhoneNumber = `+44 (0) ${ukPhoneCommonPart}`;
export const ukPhoneNumberWithoutPrefix = `0${ukPhoneCommonPart}`;
```